sdk/opendp/smartnoise/metadata/collection.py

```diff
@@ -220,17 +222,18 @@ def load_table(self, schema, table, t):
         rows_exact = int(t["rows_exact"]) if "rows_exact" in t else None
         row_privacy = bool(t["row_privacy"]) if "row_privacy" in t else False
         max_ids = int(t["max_ids"]) if "max_ids" in t else 1
-        sample_max_ids = bool(t["sample_max_ids"]) if "sample_max_ids" in t else None
-        use_dpsu = bool(t["use_dpsu"]) if "use_dpsu" in t else None
-        clamp_counts = bool(t["clamp_counts"]) if "clamp_counts" in t else None
+        sample_max_ids = bool(t["sample_max_ids"]) if "sample_max_ids" in t else True
+        use_dpsu = bool(t["use_dpsu"]) if "use_dpsu" in t else False
```

Comment:
this looks like a bad merge?

Reply:
No, it's all there if you expand.  I had to add censor_dims, since that was used in the Spark Private Synopsis, and was previously only available in PrivateReaderOptions, not metadata

Reply:
It's good; you can see all the properties if you expand

Reply:
makes sense, the change from None to True / False is what confused me